library/src/lib/menu/menu-list.directive.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { MenuListDirective } from './menu-list.directive';
+
+describe('MenuListDirective', () => {
+    it('should create an instance', () => {
```

Comment:
is there a way we can add more test than just instantiating the directive?

Reply:
all it does is add a class to the host

Reply:
can we check that? otherwise the test looks like just targeting simple code coverage.